exercises/starknet/basics/starknet1.cairo

```diff
@@ -1,28 +1,27 @@
 // starknet1.cairo
-// Address all the TODOs to make the tests pass!
-// Execute `starklings hint starknet1` or use the `hint` watch subcommand for a hint.
+// Starkling, Joe, is writing a really simple contract.
```

Comment:
I don't get why this file would be modified. The first approach was extremely simple and helped discover the following notions without too much complexity

- constructors & storage variable
- Modify contracts with external functions
- Read from contracts with view functions
- Learn how to create mappings

This new approach kinda does the same, but I find it a bit hard from starknet3 to starknet4

Reply:
But it was not really a contract :(

Reply:
So the new flow is like,
1. Adding endpoints
2. Using storage (and ContractAddress intro)
3. and 4. Finally getting serious and writing more logic with authentication.